### ISSUE TITLE
error messages in Messages window

### ISSUE THREAD

**Issue by donald-e-boyce** [HEXRD/hexrdgui#1331]
This error comes up freuqently.

```
AttributeError: 'NoneType' object has no attribute 'degrees'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/indexing/run.py", line 150, in ome_maps_selected
    config = create_indexing_config()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/indexing/create_config.py", line 52, in create_indexing_config
    config.set('material:tth_width', np.degrees(material.planeData.tThWidth))
TypeError: loop of ufunc does not support argument 0 of type NoneType which has no callable degrees method
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 481, in update_overlays
    self.draw_overlay(overlay)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 284, in draw_overlay
    for axis, data in self.overlay_axes_data(overlay):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/image_canvas.py", line 230, in overlay_axes_data
    if not overlay.data:
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/overlay.py", line 281, in data
    self._data |= self.generate_overlay()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/overlays/rotation_series_overlay.py", line 193, in generate_overlay
    sim_data = instr.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 1622, in simulate_rotation_series
    results[det_key] = panel.simulate_rotation_series(
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/instrument.py", line 3380, in simulate_rotation_series
    full_hkls = xrdutil._fetch_hkls_from_planedata(plane_data)
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/xrdutil/utils.py", line 1056, in _fetch_hkls_from_planedata
    return np.hstack(pd.getSymHKLs(withID=True)).T
  File "<__array_function__ internals>", line 180, in hstack
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/numpy/core/shape_base.py", line 345, in hstack
    return _nx.concatenate(arrs, 1)
  File "<__array_function__ internals>", line 180, in concatenate
ValueError: need at least one array to concatenate
AttributeError: 'NoneType' object has no attribute 'degrees'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/indexing/run.py", line 150, in ome_maps_selected
    config = create_indexing_config()
  File "/Users/deboyce/miniconda3/envs/hexrdgui/lib/python3.10/site-packages/hexrd/ui/indexing/create_config.py", line 52, in create_indexing_config
    config.set('material:tth_width', np.degrees(material.planeData.tThWidth))
TypeError: loop of ufunc does not support argument 0 of type NoneType which has no callable degrees method
```

**Comment by psavery** [comment]
I see. Maybe the root cause is the call to `np.degrees(material.planeData.tThWidth)` when the tth width is `None`. We might just need to add a check for that.

**Comment by psavery** [comment]
This was fixed in #1340.
